src/components/autosuggest/_macro-options.md

```diff
@@ -1,19 +1,20 @@
+| ariaOneResult       | string                               | true     | Aria message to tell the user there is only one suggestion left                                                                                                              |
+| ariaNResults        | string                               | true     | Aria message to tell the user how many suggestions are left                                                                                                                  |
+| ariaLimitedResults  | string                               | true     | Aria message to tell the user if the results have been limited and what they are limited to                                                                                  |
+| moreResults         | string                               | true     | Aria message to tell the user to continue to type to refine suggestions                                                                                                      |
+| noResults           | string                               | true     | message to tell the user there are no results                                                                                                                                |
+| tooManyResults      | string                               | false    | message to tell the user there are too many results to display and the user should refine the search. This is only required when using the address index api                 |
+| typeMore            | string                               | true     | message to encourage the user to enter more characters to get suggestions                                                                                                    |
+| resultsTitle        | string                               | true     | Title of results to be displayed on screen at the top of the results                                                                                                         |
+| resultsTitleId      | string                               | true     | ID for the results title. The ID is used in the results `aria-labelledby` to provide context for the results                                                                 |
+| input               | `Input` [_(ref)_](/components/input) | true     | Configuration object for the input                                                                                                                                           |
+| language            | string                               | false    | The ISO 639-1 Code will override the default language in page. Please note that only 'en', 'cy' and 'ni' is currently supported                                              |
+| resultsThreshold    | integer                              | false    | Option to adjust the search threshold and fuzziness. Accepts a range from 0 to 1, where 0 provides the closest match and 1 allows for more distant matches. Defaults to 0.2. |
```

Comment:
Just being very nit-picky but it isn't technically an integer. Maybe "float" would be more appropriate?